test/all.test.js

```diff
@@ -169,9 +185,17 @@ function tests() {
 			this.drive.hasMany(this.folder);
 			this.folder.belongsTo(this.drive);
 
-			return this.sequelize.sync({ force: true }).bind(this)
+			var syncOptions = { force: true };
+			if (this.dynamicSchema) {
+				syncOptions.schema = this.dynamicSchema;
+			}
+			return this.sequelize.sync(syncOptions).bind(this)
```

Comment:
I struggle here. I can't figure how sequelize-hierarchy can take into account my syncOptions.schema http://docs.sequelizejs.com/class/lib/sequelize.js~Sequelize.html#instance-method-sync

Here is the SQL log : 

```
2017-07-25 09:39:04 UTC [29008-3770] yannick@yannick LOG:  statement: DROP TABLE IF EXISTS "folder_schema"."folder_ancestor";
2017-07-25 09:39:04 UTC [29009-2877] yannick@yannick LOG:  statement: DROP TABLE IF EXISTS "folders";
2017-07-25 09:39:04 UTC [29008-3771] yannick@yannick LOG:  statement: SELECT t.typname enum_name, array_agg(e.enumlabel ORDER BY enumsortorder) enum_value FROM pg_type t JOIN pg_enum e ON t.oid = e.enumtypid JOIN pg_catalog.pg_namespace n ON n.oid = t.typnamespace WHERE n.nspname = 'public' GROUP BY 1
2017-07-25 09:39:04 UTC [29009-2878] yannick@yannick LOG:  statement: DROP TABLE IF EXISTS "foldersAncestors" CASCADE;
2017-07-25 09:39:04 UTC [29008-3772] yannick@yannick LOG:  statement: DROP TABLE IF EXISTS "folders" CASCADE;
2017-07-25 09:39:04 UTC [29009-2879] yannick@yannick LOG:  statement: DROP TABLE IF EXISTS "drives" CASCADE;
2017-07-25 09:39:04 UTC [29008-3773] yannick@yannick LOG:  statement: DROP TABLE IF EXISTS "drives" CASCADE;
2017-07-25 09:39:04 UTC [29009-2880] yannick@yannick LOG:  statement: CREATE TABLE IF NOT EXISTS "schematest"."drives" ("id"   SERIAL , "name" VARCHAR(255), "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY ("id"));
2017-07-25 09:39:04 UTC [29008-3774] yannick@yannick LOG:  statement: SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'drives' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;
2017-07-25 09:39:04 UTC [29009-2881] yannick@yannick LOG:  statement: DROP TABLE IF EXISTS "folders" CASCADE;
2017-07-25 09:39:04 UTC [29008-3775] yannick@yannick LOG:  statement: CREATE TABLE IF NOT EXISTS "schematest"."folders" ("id"   SERIAL , "name" VARCHAR(255), "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, "hierarchyLevel" INTEGER, "parentId" INTEGER REFERENCES "folders" ("id") ON DELETE RESTRICT ON UPDATE CASCADE, "driveId" INTEGER REFERENCES "drives" ("id") ON DELETE SET NULL ON UPDATE CASCADE, PRIMARY KEY ("id"));
2017-07-25 09:39:04 UTC [29008-3776] yannick@yannick ERROR:  relation "folders" does not exist
2017-07-25 09:39:04 UTC [29008-3777] yannick@yannick STATEMENT:  CREATE TABLE IF NOT EXISTS "schematest"."folders" ("id"   SERIAL , "name" VARCHAR(255), "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, "hierarchyLevel" INTEGER, "parentId" INTEGER REFERENCES "folders" ("id") ON DELETE RESTRICT ON UPDATE CASCADE, "driveId" INTEGER REFERENCES "drives" ("id") ON DELETE SET NULL ON UPDATE CASCADE, PRIMARY KEY ("id"));
2017-07-25 09:39:04 UTC [29009-2882] yannick@yannick LOG:  statement: UPDATE "schematest"."folders" SET "parentId"=NULL,"updatedAt"='2017-07-25 09:39:04.857 +00:00' WHERE "parentId" IS NOT NULL

```

@overlookmotel any ideas ?